### PR TITLE
Feature/add sort keys

### PR DESCRIPTION
**Pull request by ntrepid8**
This pull-request adds the `sort_keys` argument to the `dump` and `dumps` functions.  It is set to `False` by default.  Behavior is similar to `sort_keys` in the `json.dump` and `json.dumps` functions.

### CODE REVIEW

File: pytoml/writer.py [hunk 1]
```diff
@@ -94,7 +97,8 @@ def dump(fout, obj):
             else:
                 fout.write('[{}]\n'.format(section_name))
 
-        for k in table:
+        table_keys = sorted(table.keys()) if sort_keys is True else table.keys()
```
Comment (avakar) [inline pytoml/writer.py]: Please remove the `is True` part. There's a section about it in https://www.python.org/dev/peps/pep-0008/.

File: pytoml/writer.py [hunk 2]
```diff
@@ -6,9 +6,9 @@
     unicode = str
 
 
-def dumps(obj):
+def dumps(obj, sort_keys=False):
     fout = io.StringIO()
-    dump(fout, obj)
+    dump(fout, obj, sort_keys)
```
Comment (avakar) [inline pytoml/writer.py]: Could we add `sort_keys=` here, since we're forwarding to a keyword parameter?

**Comment by avakar** [comment]
Thank you for the pull request! I'll happily merge, but I'd like those two tiny changes made if you could. Also, would you mind squashing the commits into a single one?

**Comment by ntrepid8** [comment]
Thanks for the feedback.  I'll submit an update soon.

**Comment by ntrepid8** [comment]
This PR is superseded by #11.

**Pull request closed without merging**